R/system.R

```diff
@@ -2981,10 +2981,18 @@ load_fred <- function(series_id, date_start = "", date_end = "", password) {
   } else {
     date_end <- lubridate::ymd(date_end)
   }
-  fredr::fredr(
-    series_id = series_id,
-    observation_start = lubridate::ymd(date_start),
-    observation_end = date_end
-  )
+  # date_start is an optional parameter, so if it's not specified, execute the query without the start_date.
```

Comment:
Update the comment about date_start.
```
#'@param date_start - Start Date for the query. This is required field.
#'                    When it's called from Exploratory Desktop, this date_start is always passed since it has required input field validation.
```